packages/utils/src/lib/reports/__snapshots__/report-diff-unchanged.md

```diff
@@ -4,11 +4,11 @@
 
 ## 🏷️ Categories
 
-|🏷️ Category|⭐ Score|
+|🏷️ Category|⭐ Previous score|
 |:--|:--:|
-|Performance|🟢 **92**|
-|Bug prevention|🟡 **68**|
-|Code style|🟡 **54**|
```

Comment:
@BioPhoton @vmasek Seriously, this looks OK to you? :sweat_smile: 

**Please don't blindly replace snapshots and blindly approve PRs.** :see_no_evil: 

The whole point of snapshots is to catch unexpected regressions. But it relies on developers and reviewers _looking at them_ and judging that either:
- _"Yes, the output changing this way is expected and intentional."_
- _"No, this is unexpected, the changes must've broken something that previously worked. Thank god we have tests to catch these regressions."_ 

I can't imagine that you actually thought this is what an unchanged Categories table should look like, so my conclusion is that **neither of you really looked**. :disappointed:


![image](https://github.com/code-pushup/cli/assets/34691111/3ec96bc3-731a-43a3-a6e0-606ef6762937)

Reply:
@BioPhoton you broke this 😢 and I approve it 😓
Great example of my latest retro AP, I'll be much more careful with my future approvals

Reply:
Fixed in #734